src/service/cia301/co_csdo.c

```diff
@@ -528,7 +522,7 @@ static CO_ERR COCSdoDownloadSubBlock_Request( CO_CSDO *csdo ) {
     CO_ERR    result = CO_ERR_SDO_SILENT;
     uint32_t  ticks;
     uint32_t width;
-    int n;
+    uint32_t n;
```

Comment:
so long as you're changing the type this is a great time to change the name too. (Obligatory could this variable be const?)

Reply:
done

Reply:
Thanks!